docker/boot_notebook.bash

```diff
@@ -49,6 +36,20 @@ cat > .jupyter_config.json <<EOF
     },
     "VoilaConfiguration" : {
         "enable_nbextensions" : true
+    },
+    "ServerApp": {
```

Comment:
MINOR: as a reference, could you please add somewhere a comment with the link where the doc of this config is? thx